src/components/DpModal/DpModal.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import type { Meta, StoryObj } from '@storybook/vue'
+import DpModal from './'
+
+const meta: Meta<typeof DpModal> = {
+    component: DpModal,
+    title: 'Components/Modal',
+    render: (args) => ({
+        components: {
+            DpModal
+        },
+        setup() {
+            return { args }
+        },
+        template: `
+          <div>
+            <a
+                href="#openModal"
+                id="openModal"
+                @click.stop.prevent="$refs.openModal.toggle()"
+                role="button">
+              Open Modal
+            </a>
+            <dp-modal id="openModal" ref="openModal" v-bind="args">
+              <template v-slot:header>
+                Header Title
+              </template>
+              <template>
+                <p>
+                  Template Code & Text
+                </p>
+              </template>
+            </dp-modal>
```

Comment:
We could apply the same changes here that i suggested above.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/538/commits/11ce140eabcfc7067479d1c24cb36c95236ae48f